app/js/controllers/main.js

```diff
@@ -52,6 +52,10 @@ export default class MainController extends BaseController {
       });
 
     this[p.server].on('login', () => this[p.subscribeToNotifications]());
+    this[p.server].on('push-message', (message) => {
+      // if we're in "speaking reminders" mode (which is "always", currently)
+      this[p.speechController].speak(`${message.title}: ${message.body}`);
+    });
```

Comment:
note: this will work only if a window is still opened... we don't have access to the SpeechSynthesis api from the service worker :(